uECC.h

```diff
@@ -35,6 +35,13 @@ faster somewhat faster, but increases the code size. */
     #define uECC_SQUARE_FUNC 0
 #endif
 
+/* uECC_VLI_NATIVE_LITTLE_ENDIAN - If enabled (defined as nonzero), this will enable native
+little-endian format for all keys passed in and out of the public API. This will *only* work
+on native little-endian processors. */
```

Comment:
Please add a comment about the purpose of using this macro, eg "Enabling this option will reduce stack usage" or something.